testsuite/features/secondary/srv_virtual_host_manager.feature

```diff
@@ -24,6 +24,7 @@ Feature: Virtual host manager web UI
 
   Scenario: Create Virtual Host
     When I follow the left menu "Systems > Virtual Host Managers"
+    And I copy vCenter configuration file on server
```

Comment:
is the vcenter file necessary for the datepicker fix? I think it's separate right? Maybe open a new PR or add an explanation why this is needed here

Reply:
Ho no, something wrong with my PR. I will create a mew one...